Libs/MRML/Widgets/qMRMLLayoutManager.cxx

```diff
@@ -260,6 +263,10 @@ QWidget* qMRMLLayoutSliceViewFactory::createViewFromNode(vtkMRMLAbstractViewNode
   sliceWidget->setMRMLScene(this->mrmlScene());
   sliceWidget->setSliceLogics(this->sliceLogics());
   this->sliceLogics()->AddItem(sliceWidget->sliceLogic());
+  for (int i = 0; i < this->layoutManager()->globalPauseRenderCount(); ++i)
```

Comment:
This should be done for all new views (not just for 3D and slice views). Could we do it at one place, right after we creating the view, in qMRMLLayoutViewFactory::onViewNodeAdded?